docs/reference/cluster/stats.asciidoc

```diff
@@ -1307,6 +1307,125 @@ Number of selected nodes using the distribution flavor and file type.
 =====
 ====
 
+`snapshots`::
+(object)
+Contains statistics about the <<snapshot-restore,snapshot>> activity in the cluster.
++
+.Properties of `snapshots`
+[%collapsible%open]
+=====
+
+`current_counts`:::
+(object)
+Contains statistics which report the numbers of various ongoing snapshot activities in the cluster.
++
+.Properties of `current_counts`
+[%collapsible%open]
+======
+`snapshots`:::
```

Comment:
I wonder if it'd make sense to extend also the `_cat/snapshots/<repository>` API with any new of these counts that you think may be significant enough to be easily accessible there?

Reply:
I think you already get enough useful detail in this API - for ongoing snapshots you can see the shard-level progress. But this API shows one row per snapshot so it's not obvious to me how you'd show info about ongoing deletions or clones there.

In principle we could add it to `GET _cat/repositories` I suppose, but I think it's not worth the effort right now.